### PR TITLE
ops/storage: add block device naming conventions

### PR DESCRIPTION
**Pull request by Integral-Tech**

### CODE REVIEW

File: docs/ops/storage/filesystem.md [hunk 1]
```diff
@@ -51,6 +51,85 @@
...
+通过 SCSI 驱动子系统支持的接口连接的光盘驱动器，其名称以 `sr` 开头。之后是一个从 `0` 开始的数字，表示设备号。比如 `sr0` 表示第一个发现的设备，`sr1` 表示第二个发现的设备，以此类推。
+udev 也提供到 `/dev/sr0` 的软链接，名为 `/dev/cdrom`。软链接的名称 `/dev/cdrom` 与驱动器支持的光盘类型和插入的介质无关。
+
+例如：
+
+- `/dev/sr0` - 光盘驱动器 `0`，第一个发现的光盘驱动器。
+- `/dev/sr4` - 光盘驱动器 `4`，第五个发现的光盘驱动器。
+- `/dev/cdrom` - 到 `/dev/sr0` 的符号链接。
+
+### VirtIO (虚拟磁盘) {#virtio}
```
Comment (taoky) [inline docs/ops/storage/filesystem.md:111]: I would prefer:

```suggestion
### VirtIO（虚拟磁盘） {#virtio}
```

**Review by taoky — APPROVED** [review]

**Comment by taoky** [comment]
@iBug Could you take a look to make sure nothing is wrong here?

File: docs/ops/storage/filesystem.md [hunk 2]
```diff
@@ -51,6 +51,85 @@
...
+通过 SCSI 驱动子系统支持的接口连接的光盘驱动器，其名称以 `sr` 开头。之后是一个从 `0` 开始的数字，表示设备号。比如 `sr0` 表示第一个发现的设备，`sr1` 表示第二个发现的设备，以此类推。
+udev 也提供到 `/dev/sr0` 的软链接，名为 `/dev/cdrom`。软链接的名称 `/dev/cdrom` 与驱动器支持的光盘类型和插入的介质无关。
+
+例如：
+
+- `/dev/sr0` - 光盘驱动器 `0`，第一个发现的光盘驱动器。
+- `/dev/sr4` - 光盘驱动器 `4`，第五个发现的光盘驱动器。
+- `/dev/cdrom` - 到 `/dev/sr0` 的符号链接。
+
+### VirtIO（虚拟磁盘） {#virtio}
```
Comment (iBug) [inline docs/ops/storage/filesystem.md:111]: 这个地方是不是写明“VirtIO 块设备”比较好？毕竟还有 VirtIO SCSI 控制器这种东西

Comment (Integral-Tech) [reply docs/ops/storage/filesystem.md]: Done

**Pull request merged**